frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -98,38 +99,59 @@ function createFormGroup(controlId, formLabel, inputType, placeholder, ref) {
   }
 
   return (
-    <Form.Group controlId={controlId}>
+    <>
       <Form.Label>{formLabel}</Form.Label>
-        {formControl}
-    </Form.Group>
+      <Form.Group controlId={controlId}>
+          {formControl}
+        {/* Temporary instructions until fix Issue #52 */}
+      </Form.Group>
+    </>
   )
 }
 
 /**
- * Component corresponding to add trips modal.
+ * Component corresponding to the save trips modal.
+ *
+ * This component "acts" as a parent of the (non-existant) AddTripModal and
+ * EditTripModal components. The only differences in the implementation between
+ * the two fake components are dervied from the props `title`, `tripid`, and
+ * `placeholderObj` (see below).
  *
  * @param {Object} props These are the props for this component:
  * - db: Firestore database instance.
  * - show: Boolean that determines if the add trips modal should be displayed.
  * - handleClose: The function that handles closing the add trips modal.
  * - refreshTripsContainer: Function that handles refreshing the TripsContainer
  *        component upon trip creation (Remove when fix Issue #62).
+ * - title: The title of the modal.
+ * - tripId: For adding a new trip, this will be null. For editting an existing
+ *        trip, this will the document id associated with the trip.
+ * - placeholderObj: Object containing the placeholder/default values for the
```

Comment:
same as above with placeholderobj. Can you pull stuff out of the database when tripId != null?

Reply:
This one is not so easy. I did not do it in this PR (in PR #73 but not ready for review just yet) but I have to pass a event handler down from `index.js` to trip.js` so when the 'Edit Trip' button is pressed, the edit trip modal is opened. So if I were to remove this I would have to somehow have a handler in index that would somehow have to reference I handler in save-trip-modal.js. It will be clearer when you see it but ...

tldr: This way will be a lot cleaner based on my current knowledge.